datasets/brp_azure/personen/v0.0.1.json

```diff
@@ -16,36 +16,32 @@
           "schema": {
             "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
           },
-          "burgerservicenummer": {
+          "A-nummer": {
```

Comment:
Dit is niet toegestaan: het eerste teken moet een kleine letter zijn en koppeltekens mogen überhaupt niet voorkomen. `aNummer` zou wel mogen.

Reply:
Spaties mogen trouwens ook niet.

Reply:
Fixed; verder is het nog WIP